nccl/common/nccl-common.sh

```diff
@@ -181,7 +181,7 @@ create_instance() {
                     --query "Instances[*].InstanceId" \
                     --output=text ${addl_args} 2>&1)
             create_instance_exit_code=$?
-            echo ${INSTANCE_IDS}
```

Comment:
This was not working?

Reply:
It was working, but we had one instance. Now ${INSTANCE_IDS} will have 3 instance ids separated by spaces, so added quotes

Reply:
Thanks, got it.